falcon/inspect.py

```diff
@@ -89,8 +90,9 @@ def inspect_my_router(router):
     def wraps(fn):
         if router_class in _supported_routers:
             raise ValueError(
-                'Another function is already registered'
-                ' for the router {}'.format(router_class)
+                'Another function is already registered' ' for the router {}'.format(
```

Comment:
```suggestion
                'Another function is already registered for the router {}'.format(
```

black is also not able to do these (yet?)

Reply:
I think both are able to do these with experimental options, but it is probably better to stay safe for now.